src/Logging/PsrLogger.php

```diff
@@ -54,12 +54,12 @@ class PsrLogger implements LoggerInterface
     /**
      * @codingStandardsIgnoreStart
      * @param Logger $logger The logger used to write entries.
-     * @param array $resource The
+     * @param array $resource [optional] The
      *        [monitored resource](https://cloud.google.com/logging/docs/api/reference/rest/Shared.Types/MonitoredResource)
-     *        to associate log entries with.
+     *        to associate log entries with. **Defaults to** type global.
      * @codingStandardsIgnoreEnd
      */
-    public function __construct(Logger $logger, array $resource)
+    public function __construct(Logger $logger, array $resource = ['type' => 'global'])
```

Comment:
It seems odd we allow setting a default resource here, but not in the other `Logger` class. 

Also, we allow the setting of a default `resource`, but leave out setting a default for `labels`, which is also part of the [request body](https://cloud.google.com/logging/docs/api/reference/rest/v2/entries/write).